Umbraco-Headless/Getting-Started-Cloud/The-Umbraco-Backoffice/index.md

```diff
@@ -42,12 +42,30 @@ Should you have set URL redirects you can manage them via the Redirect URL Manag
 
 ## Content
 
-In this section you will find all the content you have on your page. Each item in the tree is called a content node. Every node is made up by different fields. Each field is defined by a document type.
+In this section you will find all the content you have on your page. Each item in the tree is called a Content Node. Every node is made up by different fields. Each field is defined by a property.
+
+The content tree holds all the content nodes you have created.
+
+![Overview of the content](images/content.png)
+
+The left holds the content tree that will automatically nest your content if you have created nodes with parent-child relationships.
+
+On the left you can see what properties the highlighted Content Node has. In the above example there is a group called Main which holds a few properties. By clicking on the Info tap you will be able to see some useful information for the specific Content Node.
+
+![Overview of the Content info](images/content-info.png)
+
+Here you can see the direct link to the front-end of the specific Content Node as well as a handy history overview that shows who has done what at what time. Lastly you can see some general information such as Status, Creation Date and a direct link to the Document Type so you can edit it without leaving the current view.
```

Comment:
I'm not sure the direct link to the frontend will work on Headless projects, as it doesn't have a frontend as such.

Reply:
As @sofietoft mentions the links won't work. I'm going to see if there's a way we can hide them as it doesn't make sence to show them when they don't work.

Reply:
Decidid to just disable the links and still show the url as it might be usefull to now when requesting through the API (`https://cdn.umbraco.io/url?url={url}`)

Reply:
Awesome @rasmusjp 😄 
I'll make sure the wording in the docs is changed a bit 💪